src/Symfony/Component/Console/Command/Command.php

```diff
@@ -273,6 +275,10 @@ public function run(InputInterface $input, OutputInterface $output): int
 
         $input->validate();
 
+        if (isset($inputDefinition)) {
```

Comment:
why the isset?
```suggestion
        if ($inputDefinition) {
```

Reply:
As @GromNaN has already explained, `$inputDefinition` is defined within a try...catch:
```
try {
    $inputDefinition = $this->getDefinition();
    $input->bind($inputDefinition);
} catch (ExceptionInterface $e) {
    if (!$this->ignoreValidationErrors) {
        throw $e;
    }
}
```
Since `getDefinition` can also throw an exception (if the parent constructor call is missing), moving it outside the try...catch makes little sense.